src/adapter/src/session.rs

```diff
@@ -1327,3 +1339,12 @@ pub struct WriteOp {
     /// The data rows.
     pub rows: Vec<(Row, Diff)>,
 }
+
+/// Whether a transaction requires linearization.
+#[derive(Debug)]
+pub enum RequireLinearization {
+    /// Linearization is not required.
+    Required,
+    /// Linearization is required.
+    NotRequired,
```

Comment:
The comments are switched?

Reply:
Oops. I'll post a fix, thanks for the heads up.